src/Raven.Server/Documents/ETL/Relational/RelationalDocumentTransformer.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Jint;
+using Jint.Native;
+using Jint.Runtime.Interop;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Server.Documents.ETL.Stats;
+using Raven.Server.Documents.Patch;
+using Raven.Server.Documents.TimeSeries;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.ETL.Relational;
+
+internal abstract class RelationalDocumentTransformer<TRelationalConnectionString, TRelationalEtlConfiguration>  : EtlTransformer<ToRelationalItem, RelationalTableWithRecords, EtlStatsScope, EtlPerformanceOperation>
```

Comment:
```suggestion
internal abstract class RelationalDatabaseDocumentTransformer<TRelationalConnectionString, TRelationalEtlConfiguration>  : EtlTransformer<ToRelationalItem, RelationalTableWithRecords, EtlStatsScope, EtlPerformanceOperation>
```


In general, I'd rather go with "relational database" prefix instead of just "relational"

Reply:
Also `Base` suffix would be nice to indicate it's an abstract class

Reply:
done